package.json

```diff
@@ -68,6 +68,7 @@
     "eslint-plugin-node": ">=9.1.0",
     "eslint-plugin-promise": ">=4.2.1",
     "eslint-plugin-standard": ">=4.0.0",
+    "typescript": ">=3.8",
```

Comment:
I don't know the version that would actually be necessary. I just picked the latest.

Reply:
Yeah, I think that this is reasonable 👍